src/main.c

```diff
@@ -435,6 +435,9 @@ static void events_cb(void *arg)
         g_tap_tempo[actuator_id].time = 0;
         g_tap_tempo[actuator_id].max = 0;
         g_tap_tempo[actuator_id].state = TT_INIT;
+
+        //clear assignment mode
+        g_current_assignment[actuator_id]->mode = CC_MODE_MOMENTARY;
```

Comment:
What is the purpose of this statement? it's not clearing but setting momentary mode

Reply:
I thought this way I would set the actuator to its init state